### ISSUE TITLE
fig. 2

### ISSUE THREAD

**Issue by tjmahr** [tjmahr/2015_Coartic_Paper#2]
Plot fig. 2 in empirical logits but use the proportion scale on the label. That way, standard errors are calculated in e-logits in both plots.
